sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -1190,4 +1201,141 @@ public HttpPipeline getHttpPipeline() {
                 objectPagedResponse.getValue().getContinuationToken(),
                 objectPagedResponse.getDeserializedHeaders()));
     }
+
+    //endregion Query APIs
+
+    //region Event Route APIs
+
+    /**
+     * Create an event route.
+     * @param eventRouteId The Id of the event route to create.
+     * @param eventRoute The event route to create.
+     * @return An empty mono.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Void> createEventRoute(String eventRouteId, EventRoute eventRoute)
```

Comment:
Is `EventRoute` under the impl package? If yes, then it will need to be moved out of it and into `/models` -> the pipeline will complain that impl classes shouldn't be referenced in public APIs